google-ads.yaml

```diff
@@ -1,14 +1,37 @@
+-# OAuth2 configuration
+-###############################################################################
+-# The below configuration parameters are used to authenticate using the       #
+-# recommended OAuth2 flow. For more information on authenticating with OAuth2 #
+-# see: https://developers.google.com/google-ads/api/docs/oauth/overview       #
+-###############################################################################
 developer_token: INSERT_DEVELOPER_TOKEN_HERE
 client_id: INSERT_OAUTH2_CLIENT_ID_HERE
 client_secret: INSERT_OAUTH2_CLIENT_SECRET_HERE
 refresh_token: INSERT_REFRESH_TOKEN_HERE
-# Required for manager accounts only: Specify the login customer ID used to
-# authenticate API calls. This will be the customer ID of the authenticated
-# manager account. It should be set without dashes, for example: 1234567890
-# instead of 123-456-7890. You can also specify this later in code if your
-# application uses multiple manager account + OAuth pairs.
+
+-# Login customer ID configuration
+-###############################################################################
+-# Required for manager accounts only: Specify the login customer ID used to   #
+-# authenticate API calls. This will be the customer ID of the authenticated   #
+-# manager account. It should be set without dashes, for example: 1234567890   #
+-# instead of 123-456-7890. You can also specify this later in code if your    #
+-# application uses multiple manager account + OAuth pairs.                    #
+-###############################################################################
 login_customer_id: INSERT_LOGIN_CUSTOMER_ID_HERE
 
+-# Service Account configuration
+-###############################################################################
+-# To authenticate with a service account add the appropriate values to the    #
+-# below configuration parameters and remove the four OAuth credentials above. #
+-# The "path_to_private_key_file" value should be a path to your local private #
+-# key json file, and "delegated_account" should be the email address that is  #
+-# being used to impersonate the credentials making requests. for more         #
+-# information on service accounts, see:                                       #
```

Comment:
might want to add a pointer to the README about removing the Installed Application Flow in order to use these parameter.

Reply:
Done